test/common/link-patterns.json

```diff
@@ -18,7 +18,7 @@
     },
     {
         "label": "changed pixels",
-        "pattern": "Differences in pixel test ([A-Za-z0-9.-]+)$",
+        "pattern": "Pixel test ([A-Za-z0-9.-]+) failed:",
```

Comment:
This change also needs to go into https://github.com/cockpit-project/bots/blob/main/lib/s3-html/log.html#L317 . I'll do that after this got reviewed.

Reply:
What about adding "_" add the same time?

Reply:
Sure! Done both in https://github.com/cockpit-project/bots/pull/8085